### ISSUE TITLE
--with-llvm brew installation failed

### ISSUE THREAD

**Issue by akustovbanuba** [doxygen/doxygen#6581]
There is a doxygen installation problem using brew as follows:
brew install doxygen --with-llvm

Here is an output:
```
==> Downloading https://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.14.src.tar.gz
Already downloaded: /Users/antonkustov/Library/Caches/Homebrew/downloads/642be3695a5ba4ea5ce13e93f272cbbca18ddbc21a3fd377867056cf16592aee--doxygen-1.8.14.src.tar.gz
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/doxygen/1.8.14 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOS
==> make
Last 15 lines from /Users/antonkustov/Library/Logs/Homebrew/doxygen/02.make:
/tmp/doxygen-20181029-57508-jw3eg2/doxygen-1.8.14/src/clangparser.cpp:187:23: error: member access into incomplete type 'clang::tooling::CompilationDatabase'
          command = db->getCompileCommands(fileName);
                      ^
/usr/local/opt/llvm/include/clang/Tooling/Tooling.h:68:7: note: forward declaration of 'clang::tooling::CompilationDatabase'
class CompilationDatabase;
```

macOS v. 10.13.6
homebrew 1.8.0
cmake 3.12.3

**Comment by aflp91** [comment]
Same error here …

System Version: macOS 10.13.6 (17G3025)
Homebrew 1.8.3-27-g2063a8a
cmake version 3.13.0

**Comment by dvj** [comment]
This was fixed in https://github.com/doxygen/doxygen/pull/740 but there hasn't been a release since then. Use `--HEAD` on the brew line to fix until there is a new release.

**Comment by aflp91** [comment]
@dvj It ran … Thank you!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
